styles.css

```diff
@@ -93,3 +93,163 @@ nav {
 /* frontpage */
 
 /* -------------------------------- */
+
+/* projectssection */
+
+.project1 {
+  background-color: #fff;
+  margin: 0 3% 0 3%;
+  border-radius: 20px;
+  border: 1px solid #dfe1e6;
+  margin-bottom: 30%;
+  padding-bottom: 3%;
+}
```

Comment:
- [ ] Although the content of each project looks good :+1:, it's important to note that as a project requirement, each project card must have a `display:flex` property. Please make the necessary adjustments to adhere to this requirement.